### DISCUSSION TITLE
Differential along trajectory

### DISCUSSION THREAD

**Discussion by AnjaliC4** [GreenleafLab/ArchR#1495] [Questions / Documentation] [answered]
Hi,

I have a few questions about trajectory analysis.
 
1. One of the important trajectory analysis is to identify differential features between two conditions(case/control) along the trajectory as implemented by Monocle3 and slingshot + Trade-seq. 
https://cole-trapnell-lab.github.io/monocle3/docs/differential/
How do you suggest one can do this in ArchR? Is it possible to implement wilcox test to comapre the two conditions along the trajectory from the output of getTrajectory, for example?

2. Is there a way to add column names (names of cluster lineages) on plotTrjectoryHeatmap(). Otherwise, its not possible to map the genes to celltypes present in lineage. 

3. In Slingshot trajectories in ArchR, there are always more than one Curves for a trajectory that are added to cellColData. Which ones are to be used? For example, Traj.Curve1, Traj.Curve2, etc

Thanks

**Comment by rcorces** [comment, marked as answer]
1. I'm not sure what you mean by "differential features between two conditions(case/control) along the trajectory". This seems most reasonable to do at specific pseudotime points along the trajectory. For example, "in the first part of the trajectory, how does condition A differ from condition B". If that is what you want, the trajectory is just added as another column in `cellColData` so you just need to identify the cells of interest and do a differential comparison as normal.
2. No, the columns in a trajectory heatmap are not cells, they are pseudotime points
```
> head(colnames(trajMM))
[1] "T.0_1" "T.1_2" "T.2_3" "T.3_4" "T.4_5" "T.5_6"
```
3. It is not always the case that you get multiple curves (the tutorial data gives one curve). You will need to decide which curve is best for your analysis.
